repos/system_upgrade/supplements/topics/systeminfo.py

```diff
@@ -0,0 +1,5 @@
+from leapp.topics import Topic
+
+
+class SystemInfoTopic(Topic):
```

Comment:
Remove this because already defined in leapp-repository?

Reply:
yep, remove